static/app/components/smartSearchBar/index.tsx

```diff
@@ -281,6 +246,10 @@ type Props = WithRouterProps & {
    * Indicates the usage of the search bar for analytics
    */
   searchSource?: string;
+  /**
+   * Whether or not to highlight unsupported tags red
+   */
+  showUnsupportedTags?: boolean;
```

Comment:
nit: think this would be a little clearer if it were named `highlightUnsupportedTags`